pkg/miekg/miekg.go

```diff
@@ -1,3 +1,6 @@
+//go:build !mock
```

Comment:
What's the uses for the build constraints here? Do we need to pass any flags to build mocks while testing?

Reply:
rather, does this need to be excluded from any builds during testing?